wiring/inc/spark_wiring_system.h

```diff
@@ -434,20 +434,28 @@ class SystemClass {
 
     static void enterSafeMode(SystemResetFlags flags = SystemResetFlags());
 
-#if SYSTEM_HW_TICKS
+#if HAL_PLATFORM_SYSTEM_HW_TICKS
     static inline uint32_t ticksPerMicrosecond() {
         return SYSTEM_US_TICKS;
     }
 
     static inline uint32_t ticks() {
         return SYSTEM_TICK_COUNTER;
     }
+#else
+    static inline uint32_t ticksPerMicrosecond() {
+        return 1;
```

Comment:
Shouldn't this still be `SYSTEM_US_TICKS`?

Reply:
I guess it does seem like this another magic number, but on the off chance we define a new platform without defining SYSTEM_US_TICKS properly, this will ensure the API ticksPerMicrosecond() is not affected.